src/mapping/cellranger_multi/script.py

```diff
@@ -197,7 +208,31 @@ def resolve_input_directories_to_fastq_paths(input_paths: list[str]) -> list[Pat
 
     return input_paths
 
-def process_params(par: dict[str, Any]) -> str:
+def make_paths_absolute(par: dict[str, Any], config: Path | str):
+    with open(config, 'r', encoding="utf-8") as open_viash_config:
+        config = yaml.safe_load(open_viash_config)
+
+    arguments = {
+        arg["name"].removeprefix("-").removeprefix("-"): arg
+        for group in config["functionality"]["argument_groups"]
+        for arg in group["arguments"]
+    }
+    file_arguments = {arg_name: arg for arg_name, arg in arguments.items()
+                      if arg['type'] == 'file'}
+    for file_arg_name, arg_options in file_arguments.items():
+        if not par.get(file_arg_name):
+            continue
+        elif arg_options['multiple']:
+            new_arg = []
+            for file_path in par[file_arg_name]:
+                logger.info('Making path %s absolute', file_path)
+                new_arg.append(file_path.resolve())
+            par[file_arg_name] = new_arg
+        elif par.get(file_arg_name):
+            par[file_arg_name] = Path(par[file_arg_name]).resolve()   
```

Comment:
I notice that the transformation to a `Path(.)` is not happening when `multiple: true`.

I propose keeping the logic where value transformations are made in one place, e.g.:

```suggestion
        value = par[file_arg_name]
        is_list = value isinstance list
        if not is_list:
          value = [value]
        value = [Path(x).resolve() for x in value]
        if not is_list:
          value = value[0]
        par[file_arg_name] = value 
```

Alternatively, if you don't mind the inclusion of an if-else statement:

```suggestion
        def make_path_absolute(file: string) -> Path:
          return Path(file).resolve()
        if par[file_arg_name] isinstance list:
          par[file_arg_name] = [make_absolute(file) for file in par[file_arg_name]]
        else:
          par[file_arg_name] = make_absolute(par[file_arg_name])
```

(Helper function could be moved higher)